src/Connection.php

```diff
@@ -277,7 +281,7 @@ protected function getDefaultQueryGrammar()
     /**
      * @inheritdoc
      */
-    protected function getDefaultSchemaGrammar()
+    protected function getDefaultSchemaGrammar(): Schema\Grammar
```

Comment:
Isn't the addition of return type hints on non-private methods a potential BC break for inheriting classes?

Reply:
That is correct. Protected members in non-final classes have to be treated like public members when considering BC breaks.